.gitignore

```diff
@@ -37,3 +37,6 @@ build/
 
 # macOS files
```

Comment:
Are we still working on a p4_library bazel function?

Reply:
Yes.